tests/logging_tests/tests.py

```diff
@@ -483,14 +464,17 @@ class SetupConfigureLogging(SimpleTestCase):
     """
     Calling django.setup() initializes the logging configuration.
     """
-    @override_settings(
-        LOGGING_CONFIG='logging_tests.tests.dictConfig',
-        LOGGING=OLD_LOGGING,
-    )
     def test_configure_initializes_logging(self):
         from django import setup
-        setup()
-        self.assertTrue(dictConfig.called)
+        try:
+            with override_settings(
+                LOGGING_CONFIG='logging_tests.tests.dictConfig',
+            ):
+                setup()
+            self.assertTrue(dictConfig.called)
```

Comment:
The assertion could be taken out of the try block. NBD.

Reply:
True.